docs/libraries.md

```diff
@@ -34,3 +34,4 @@ We also have many great contrubutors that are maintaining extra libraries for la
     * [C (with optional Perl, Python and Ruby bindings)](https://github.com/in4lio/ev3dev-c) by @in4lio
     * [C](https://github.com/theZiz/ev3c) by @Ziz
     * [Clojure](https://github.com/annapawlicka/clj-ev3dev) by @annapawlicka
+    * [vala](https://github.com/dlech/ev3dev-lang-glib) by @dlech and @WasabiFan
```

Comment:
For vala, which actually works with any language that supports gobject introspection, the code has been merged into https://github.com/ev3dev/ev3devKit